server/src/main/java/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -1644,9 +1644,7 @@ public Network createGuestNetwork(CreateNetworkCmd cmd) throws InsufficientCapac
             if (ipv4 && Objects.isNull(startIP)) {
                 throw new CloudRuntimeException("IPv4 address range needs to be provided");
             }
-            if (ipv6 && Objects.isNull(startIPv6)) {
-                throw new CloudRuntimeException("IPv6 address range needs to be provided");
-            }
+            // if (ipv6) ignore entries
```

Comment:
```suggestion
            if (ipv6 && Objects.isNull(startIPv6)) {
                logger.info("ip ranges for network '{}' are being ignored, because ipv6 is used", name);
            }
```

Reply:
@weizhouapache like this, right?

Reply:
I do not know what is the best word to describe it ....

the behaviour is 

- if ipv6 range is passed, it is saved in database
![image](https://github.com/user-attachments/assets/be8949dd-0deb-457d-bdff-dcfc98c6e389)

- but vm gets a IP out of the ipv6 range
![image](https://github.com/user-attachments/assets/aee1271f-1ea8-4ef6-b1e5-b40a5209bfa4)

Reply:
hm, ok, I’ll give it another thought, and change.

Reply:
does it work for 4.19 ? (log4j 1.x)

Reply:
:D :face_with_head_bandage:

Reply:
```suggestion
            if (ipv6) {
                logger.info("ip range for network '%s' is specified as %s - %s", name, startIPv6, endIPv6);
            }
```

and thanks for being awake.

Reply:
String.format ?

Reply:
yeah

Reply:
```suggestion
            if (ipv6) {
                logger.info(String.format("ip range for network '%s' is specified as %s - %s", name, startIPv6, endIPv6));
            }
```
I hate me

Reply:
looks good